devito/builtins.py

```diff
@@ -123,46 +135,61 @@ def fset(f, g):
         else:
             raise NotImplementedError
 
-    lw = int(_order*sigma + 0.5)
+    try:
+        dtype = f.dtype.type
+    except AttributeError:
+        dtype = f.dtype
+
+    # TODO: Add s = 0 dim skip option
```

Comment:
leftover?

Reply:
No, still need to add that functionality.